go/v1/api/note_test.go

```diff
@@ -38,13 +39,13 @@ func TestCreateNote(t *testing.T) {
 		NoteId: "CVE-UH-OH",
 		Note:   vulnzNote(t),
 	}
-	n := &gpb.Note{}
-	if err := g.CreateNote(ctx, req, n); err != nil {
-		t.Errorf("Got err %v, want success", err)
+	createdNote := &gpb.Note{}
+	if err := g.CreateNote(ctx, req, createdNote); err != nil {
+		t.Fatalf("Got err %v, want success", err)
 	}
 
-	opt := cmp.FilterPath(func(p cmp.Path) bool { return p.String() == "Name" }, cmp.Ignore())
-	if diff := cmp.Diff(req.Note, n, opt); diff != "" {
+	createdNote.Name = ""
```

Comment:
Could you please add a TODO here with the note that we need to migrate to another tool? Like you explained in the PR description.

Reply:
Done.